Add AGENTS.md agent guidance with CLAUDE.md symlink (#102)
ultralytics/llm: README.md

```bash
npm install -g terser
terser js/chat.js -o js/chat.min.js -c -m --comments false
```

> > **Note**: You don't need to commit `chat.min.js` manually — the [Minify workflow](.github/workflows/minify.yml) rebuilds and commits it automatically on PRs that modify `js/chat.js`.

Reviewer: 📝 **LOW**: This note is too broad for forked PRs. The minify workflow is triggered on PRs touching `js/chat.js`, but its job condition skips PRs whose head repo differs from `ultralytics/llm`, so external contributors will not get an automatic commit for `js/chat.min.js`. Please qualify this as same-repository PR behavior or document what fork contributors should do.

**Suggested change:**
```suggestion
> **Note**: For same-repository PRs, you don't need to commit `chat.min.js` manually — the [Minify workflow](.github/workflows/minify.yml) rebuilds and commits it automatically when `js/chat.js` changes.
```
